src/main/webapp/index.html

```diff
@@ -18,37 +18,48 @@
 <html>
 
 <head>
-  <meta charset="UTF-8">
-  <title>CodeU 2019 Starter Project</title>
-  <link rel="stylesheet" href="/css/main.css">
-  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
- 
+    <meta charset="UTF-8">
```

Comment:
I think these changes are whitespace-only, would you be able to configure your editor for HTML to use 2 spaces (instead of 4)?